rts/Game/UI/MiniMap.cpp

```diff
@@ -1134,7 +1130,6 @@ void CMiniMap::DrawForReal(bool useNormalizedCoors, bool updateTex, bool luaCall
 		// switch to normalized minimap coords
 		if (globalRendering->dualScreenMode) {
 			glViewport(curPos.x, curPos.y, curDim.x, curDim.y);
-			glScalef(curDim.x * globalRendering->pixelX, curDim.y * globalRendering->pixelY, 1.0f);
```

Comment:
Does not make sense to adjust for main view screen resolution if dual view is present on separate screen.